schema/block.proto

```diff
@@ -3,35 +3,44 @@ package iroha.protocol;
 import "commands.proto";
 import "primitive.proto";
 
+message Header {
+  uint64             created_time    = 1;
```

Comment:
Does it support 64-bit timestamps? Since they are used here https://github.com/hyperledger/iroha/blob/develop/irohad/ordering/queue.cpp#L56